draft/spec/index.html

```diff
@@ -1296,172 +1298,162 @@ <h2>Moab in an OCFL Object</h2>
-    "e64db0...500": [ "v0003/content/data/metadata/descMetadata.xml" ],
-    "05fa51...818": [ "v0003/content/data/metadata/rightsMetadata.xml" ],
-    "d70dd8...5ad": [ "v0003/content/data/metadata/events.xml" ],
-    "509a2d...dc6": [ "v0003/content/data/metadata/identityMetadata.xml" ],
-    "548066...893": [ "v0003/content/data/metadata/versionMetadata.xml" ],
-    "93884e...aae": [ "v0003/content/data/metadata/contentMetadata.xml" ],
-    "4c5ab4...b02": [ "v0003/content/data/metadata/embargoMetadata.xml" ]
-}
+  {
+    "digestAlgorithm": "sha512",
+    "head": "v3",
+    "id": "druid:bj102hs9687",
+    "contentDirectory": "data",
+    "manifest": {
+        "98114a...588": [ "v0001/data/content/eric-smith-dissertation-augmented.pdf" ],
+        "7f3d87...15b": [ "v0001/data/content/eric-smith-dissertation.pdf" ],
+        "6d19f0...064": [ "v0001/data/metadata/technicalMetadata.xml" ],
+        "6e4be4...375": [ "v0001/data/metadata/provenanceMetadata.xml" ],
+        "d8a319...d0f": [ "v0001/data/metadata/descMetadata.xml" ],
+        "de823a...acc": [ "v0001/data/metadata/rightsMetadata.xml" ],
+        "080617...40c": [ "v0001/data/metadata/identityMetadata.xml" ],
+        "e15267...58d": [ "v0001/data/metadata/versionMetadata.xml" ],
+        "0d9e0b...9a2": [ "v0001/data/metadata/contentMetadata.xml" ],
+        "dd9289...31d": [ "v0001/data/metadata/relationshipMetadata.xml" ],
+        "7519c5...63f": [ "v0002/data/metadata/provenanceMetadata.xml" ],
+        "abda4c...622": [ "v0002/data/metadata/workflows.xml" ],
+        "76549e...b2b": [ "v0002/data/metadata/rightsMetadata.xml" ],
+        "bdc4d6...3b6": [ "v0002/data/metadata/events.xml" ],
+        "7b331c...f9b": [ "v0002/data/metadata/identityMetadata.xml" ],
+        "80ceac...b9c": [ "v0002/data/metadata/versionMetadata.xml" ],
+        "4853a2...fbe": [ "v0002/data/metadata/contentMetadata.xml" ],
+        "1d5090...f5f": [ "v0002/data/metadata/relationshipMetadata.xml" ],
+        "f209bf...ceb": [ "v0002/data/metadata/embargoMetadata.xml" ],
+        "dd9125...d4b": [ "v0003/data/metadata/technicalMetadata.xml" ],
+        "d9e177...477": [ "v0003/data/metadata/provenanceMetadata.xml" ],
+        "4f5908...4f5": [ "v0003/data/metadata/workflows.xml" ],
+        "e64db0...500": [ "v0003/data/metadata/descMetadata.xml" ],
+        "05fa51...818": [ "v0003/data/metadata/rightsMetadata.xml" ],
+        "d70dd8...5ad": [ "v0003/data/metadata/events.xml" ],
+        "509a2d...dc6": [ "v0003/data/metadata/identityMetadata.xml" ],
+        "548066...893": [ "v0003/data/metadata/versionMetadata.xml" ],
+        "93884e...aae": [ "v0003/data/metadata/contentMetadata.xml" ],
+        "4c5ab4...b02": [ "v0003/data/metadata/embargoMetadata.xml" ]
+    }
+    ...
 </pre>
     <p>
-        The version 1 state block would look like this. Note the absence of the manifests/ directory.
+        The version 1 state block would look like this.
```

Comment:
Instead of separating out the `manifest` and three `state` blocks into separate code examples, would it be better to simply write out the complete inventory as one example? There would then be no doubt about what else should or should not be present _AND_ we could validate it.

Reply:
Consolidated example into a single inventory file ; it's hand-rolled so I hope it validates.